.gitignore

```diff
@@ -2,4 +2,6 @@ todo.md
 
 .vscode
 
-.idea
\ No newline at end of file
+.idea
+
+.DS_Store
```

Comment:
They do not seem to appear on my side. Only DS_Store is present on the main branch, but on develop branch, on the Assignment2 folder I can't see them. Probably it will fix the ones on main when the branch will be merged.